decidim-initiatives/spec/cells/decidim/initiatives/content_blocks/highlighted_initiatives_cell_spec.rb

```diff
@@ -21,11 +19,13 @@
 
   context "when the content block has no settings" do
     it "shows 4 initiatives" do
-      expect(highlighted_initiatives).to have_selector("a.card__grid", count: 4)
+      within "#diff-for-body" do
```

Comment:
This  "within #diff-for-body" isn't necessary in this file. Can you remove it here and in the other examples of this file please?